Sources/VGSCollectSDK/Utils/Helpers/Serializers/VGSFormatSerializerProtocol.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  VGSFormatSerializerProtocol.swift
+//  VGSCollectSDK
+//
+//  Created by Dima on 25.03.2021.
+//  Copyright © 2021 VGS. All rights reserved.
+//
+
+import Foundation
+
+/// Base protocol describing Content Serialization attributes
+public protocol VGSFormatSerializerProtocol {
```

Comment:
Please check can we have some `internal` protocol to avoid adding non-VGS serializers.